apps/tai/lib/tai/events/order_updated.ex

```diff
@@ -90,6 +90,10 @@ defimpl TaiEvents.LogEvent, for: Tai.Events.OrderUpdated do
       :enqueued_at,
       event.enqueued_at && event.enqueued_at |> DateTime.to_iso8601()
     )
+    |> Map.put(
+      :last_received_at,
+      event.last_received_at && event.last_received_at |> DateTime.to_iso8601()
```

Comment:
This needs to convert monotonic to DateTime before formatting to iso in log events.

Reply:
It's converted in the actions before it gets to `OrderUpdated` e.g. https://github.com/fremantle-capital/tai/pull/203/files#diff-d0f858bf1f355c1e1a4702fbf5f2a0ca131cc0520141aaab2b2f8ca7eff975f1R22